packages/react/label/src/Label.stories.tsx

```diff
@@ -76,17 +127,25 @@ const rootClass = css({
 });
 
 const controlClass = css({
-  display: 'inline-block',
+  display: 'inline-flex',
   border: '1px solid gainsboro',
   padding: 10,
   verticalAlign: 'middle',
   margin: '0 10px',
 
-  '&::after': {
+  '&::before': {
```

Comment:
This changed the output of chromatic obviously, but was it intentional? The output doesn't read as well now.

Reply:
Ah, good eyes! No need for this change, I was fiddling with styling. Will revert!

Reply:
Story looks like this now:

https://user-images.githubusercontent.com/175330/133243671-a8d3336d-6ea2-4e75-8311-ef75052f18a4.mp4